pkg/runtime/kubeadm.go

```diff
@@ -0,0 +1,56 @@
+package runtime
+
+import (
+	"github.com/alibaba/sealer/logger"
+	"strings"
+)
+
+var (
+	ContainerdShell = `if grep "SystemdCgroup = true"  /etc/containerd/config.toml &> /dev/null; then  
+driver=systemd
+else
+driver=cgroupfs
+fi
+echo ${driver}`
+	DockerShell = `driver=$(docker info -f "{{.CgroupDriver}}")
+	echo "${driver}"`
+)
+
+func (k *KubeadmRuntime) setKubeadmAPIAndCriSocketByVersion() {
+	switch {
+	case VersionCompare(k.Metadata.Version, V1150) && !VersionCompare(k.Metadata.Version, V1200):
```

Comment:
I think we don't need though version to config cri socket, each CloudImage care about it own criconfig, so that v1.20 also can using docker as CRI